moped-editor/src/components/GridTable/Search.js

```diff
@@ -122,6 +123,28 @@ const Search = ({
     });
   };
 
+  /**
+   * Handles the submission of our search form
+   * @param {Object} e - The event object
+   */
+  const handleSearchSubmission = (event) => {
```

Comment:
pulled this up from `SearchBar` so that we could also pull the mobile search button up into this component so it is in the same container as the map switch and download button